server/src/streem/util.ts

```diff
@@ -0,0 +1,33 @@
+import { Request } from 'express';
+import { createHmac, timingSafeEqual } from 'crypto';
+import { SIGNING_KEY_SECRET } from '../routes/webhooks';
+
+export function validateWebhookRequest(req: Request): Boolean {
+    const hmac = createHmac('sha256', Buffer.from(SIGNING_KEY_SECRET, 'utf-8').toString());
```

Comment:
`SIGNING_KEY_SECRET` is already a string. Encoding it into a `Buffer` just to call `toString()` on it is superfluous. You can just use `SIGNING_KEY_SECRET` directly.